.github/workflows/ci.yaml

```diff
@@ -0,0 +1,19 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+
+permissions:
+  pull-requests: write
+  contents: write
+  id-token: write
+
+jobs:
+  build:
+    uses: Workiva/gha-dart-oss/.github/workflows/build.yaml@v0.1.5
+
+  test-unit:
+    uses: Workiva/gha-dart-oss/.github/workflows/test-unit.yaml@v0.1.5
```

Comment:
there's no dart code in this repo, other than a few tests, no need to run checks